README.md

```diff
@@ -42,14 +42,13 @@ For help in utilizing `dj.config` (added in `3.4.0`), you may access the help vi
 
 * Create an `.env` with desired development environment values e.g.
 ``` sh
-MATLAB_USER=rguzman
-MATLAB_LICENSE=IyBCRUd... # For image usage instructions see https://github.com/guzman-raphael/matlab, https://hub.docker.com/r/raphaelguzman/matlab
-MATLAB_VERSION=R2019a
+MATLAB_USER=raphael
+MATLAB_LICENSE="#\ BEGIN----...---------END" # For image usage instructions see https://github.com/guzman-raphael/matlab, https://hub.docker.com/r/raphaelguzman/matlab
+MATLAB_VERSION=R2018b
```

Comment:
Oops. I think for the changes to this file, the prior values seem to be more recent.